src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx

```diff
@@ -338,15 +339,20 @@ export class Relationships extends Component<RelationshipsProps, RelationshipsSt
         <EuiFlyoutHeader hasBorder>
           <EuiTitle size="m">
             <h2>
-              <EuiToolTip position="top" content={getSavedObjectLabel(savedObject.type)}>
+              <EuiToolTip
+                position="top"
+                content={getSavedObjectLabel(savedObject.type)}
+                anchorClassName="savedObjectsManagementRelationships__tooltipAnchor"
```

Comment:
See note above - we should only be removing classes and styles, not adding. In this case, I believe the correct solution is to use the existing utility class `oui-textBreakWord`.

But even this is not ideal - we should open an issue in OUI to make sure that we use break word by default on `OuiTitle`.